client/src/main/java/io/pravega/client/state/SynchronizerConfig.java

```diff
@@ -21,10 +21,12 @@
 @Builder
 public class SynchronizerConfig implements Serializable {
```

Comment:
should we move this to pravega's custom serialization? or is the synchronizerConfig class expected to be serialized and deserialized by user/application code too?

Reply:
Yes: https://github.com/pravega/pravega/issues/4097